Server/sprawl_calculator/wup_calculator/models.py

```diff
@@ -1,9 +1,16 @@
 from django.db import models
+from datetime import datetime
 import uuid
 # Create your models here.
 
 class WUPCalculation(models.Model):
     uuid = models.UUIDField(max_length=36, min_length=36, format='uuid', default=uuid.uuid4, unique=True)
+    resident_count_in_boundary = models.IntegerField()
+    employment_count_in_boundary = models.IntegerField()
+    raster_with_build_up_area = models.URLField()
+    raster_no_data_value = models.IntegerField()
+    raster_build_up_value = models.IntegerField()
+    vector_boundary = models.PolygonField()
```

Comment:
This shuold be a Textfield or JSON field you can review the Django documentation for one to pick but basically the data will be a GeoJSON polygon, see e.g. geojson.io